use ${LD} instead of ld, so cross compiling / linking does not break

--- mk/gcc.mk      2018-03-09 19:58:51.451543586 +0100
+++ mk/gcc.mk      2018-03-09 19:59:08.897543274 +0100
@@ -24,7 +24,7 @@
 LDFLAGS_LIB=-dynamiclib
 LDFLAGS_SONAME=-Wl,-install_name,${LIBDIR}/
 else
-PARTIALLD=ld -r --whole-archive
+PARTIALLD=${LD} -r --whole-archive
 LDFLAGS_LIB=${LDFLAGS} -shared
 #ifneq (${NAME},)
 #LDFLAGS_LIB+=-Wl,-soname,lib${NAME}.${EXT_SO}.${VERSION}

